### ISSUE TITLE
Best segmentation model that can be used along with MobilentV1 image classification model?

### ISSUE THREAD

**Issue by Anishauday** [keras-team/keras#14328]
I have already developed a image classification (4 classes) using a pretrained mobilenetv1 model. Now I wanted to do segmentation (based on different classes in the image). Which segmentation model will be the right fit? Also should I again retrain the model on top of the classification to achieve the output with segmented masks?

**Comment by sushreebarsa** [comment]
Could you please refer to this [documentation  ](https://www.tensorflow.org/tutorials/images/segmentation?hl=en) and let us know if it helps ? Thanks !

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity & assuming this issue as resolved .Please feel free to reopen if you still have concern.Thanks!
